tests/ModelTest.php

```diff
@@ -1188,6 +1190,12 @@ public function testCreateWithNullId()
         $this->assertSame(1, User::count());
     }
 
+    public function testDocumentVersion()
```

Comment:
This test class is already very long. You can create en new `DocumentVersionTest` class.

Reply:
Ok, i create a new class